src/lib/schema/feature-schema.ts

```diff
@@ -50,7 +50,13 @@ export const variantValueSchema = joi
 
 export const variantsSchema = joi.object().keys({
     name: nameType,
-    weight: joi.number().min(0).max(1000).required(),
+    weight: joi
+        .number()
+        .integer()
+        .message('Weight only supports 1 decimal')
```

Comment:
This is just to improve the error message. Otherwise it'll show that the value has to be integer which is misleading for oru users